rktlet/runtime/pod_sandbox.go

```diff
@@ -82,22 +82,7 @@ func (r *RktRuntime) RunPodSandbox(ctx context.Context, req *runtimeApi.RunPodSa
 		return nil, fmt.Errorf("waited 10s for pod sandbox to start, but it didn't: %v", k8sPodUid)
 	}
 
-	// Wait for the status to be running too, up to 10 more seconds
-	var status *runtimeApi.PodSandboxStatusResponse
-	for i := 0; i < 100; i++ {
-		status, err = r.PodSandboxStatus(ctx, &runtimeApi.PodSandboxStatusRequest{PodSandboxId: rktUUID})
-		if err == nil {
-			break
-		}
-		if st := status.GetStatus(); st == nil || st.State != runtimeApi.PodSandboxState_SANDBOX_READY {
-			continue
-		}
-		time.Sleep(100 * time.Millisecond)
-	}
-	if status.GetStatus().State != runtimeApi.PodSandboxState_SANDBOX_READY {
-		glog.Warningf("sandbox got a UUID but did not have a ready status after 10s: %v, %v", status, err)
-		return &runtimeApi.RunPodSandboxResponse{PodSandboxId: rktUUID}, fmt.Errorf("unable to get status within 10s: %v", err)
-	}
+	_ = r.waitPodSandboxStatus(ctx, id, runtimeApi.PodSandboxState_SANDBOX_READY)
```

Comment:
Shouldn't you get the error and return `&runtimeApi.RunPodSandboxResponse{PodSandboxId: rktUUID}, err` if not nil?

Reply:
@iaguis Done. Added error checks.